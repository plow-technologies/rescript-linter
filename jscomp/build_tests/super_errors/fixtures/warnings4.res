type myType = [
  | #first
  | #second(string)
  | #third
  | #fourth
]

// just a quick way to make the switch compile
@val external x: myType = "myVariable"

switch x {
  | #first => Js.log("first")
}
