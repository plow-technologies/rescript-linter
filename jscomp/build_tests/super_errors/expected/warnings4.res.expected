
  [1;33mWarning number 8[0m
  [36m/.../fixtures/warnings4.res[0m:[2m11:1-13:1[0m

   9 [2mâ”‚[0m @val external x: myType = "myVariable"
  10 [2mâ”‚[0m 
  [1;33m11[0m [2mâ”‚[0m [1;33mswitch x {[0m
  [1;33m12[0m [2mâ”‚[0m [1;33m  | #first => Js.log("first")[0m
  [1;33m13[0m [2mâ”‚[0m [1;33m}[0m
  14 [2mâ”‚[0m 

  You forgot to handle a possible case here, for example: 
  | #second(_) | #fourth | #third